src/App/App.tsx

```diff
@@ -7,21 +7,24 @@ import Banner from "./Banner/Banner";
 import LoginBoundary from "./LoginBoundary";
 import { OrganisasjonsListeProvider } from "../OrganisasjonsListeProvider";
 import { OrganisasjonsDetaljerProvider } from "../OrganisasjonDetaljerProvider";
+import { Normaltekst } from "nav-frontend-typografi";
 
 const App: FunctionComponent = () => {
   return (
     <LoginBoundary>
       <OrganisasjonsListeProvider>
         <OrganisasjonsDetaljerProvider>
-          <div className={"bakgrunnsside"}>
-            <Banner tittel={"Ditt nav arbeidsgiver"} />
-            <BrowserRouter basename={basename}>
-              <Switch>
-                <Route path="/" exact={true} component={Hovedside} />
-                {<Redirect to={"/"} />}
-              </Switch>
-            </BrowserRouter>
-          </div>
+          <Normaltekst tag={"div"}>
```

Comment:
Her kommer det 2 helt tomme div-tags rett under hverandre. Det er en fordel både for kodekvalitet og render-tid (visstnok) at man ikke nester så mye. Disse to kan byttes ut med `<div className="bakgrunnsside typo-normal">`. (typo-normal gir stylingen til Normaltekst)